www/include/configuration/configObject/connector/formConnector.php

```diff
@@ -171,13 +171,15 @@
         $cntObj = new CentreonConnector($pearDB);
         $tab = $form->getSubmitValues();
         $connectorValues = array();
-        $connectorValues['name'] = $tab['connector_name'];
-        $connectorValues['description'] = $tab['connector_description'];
+        $connectorValues['name'] = filter_var($tab['connector_name'], FILTER_SANITIZE_STRING);
+        $connectorValues['description'] = filter_var($tab['connector_description'], FILTER_SANITIZE_STRING);
+        $connectorValues['enabled'] = $tab['connector_status']['connector_status'] === 0 ? 0 : 1;
```

Comment:
are you sure it's an integer which is returned by the form ?

Reply:
form return string and db wait a number